src/Platforms/Gcc/UtestPlatform.cpp

```diff
@@ -235,7 +235,7 @@ void* (*PlatformSpecificMemset)(void*, int, size_t) = memset;
 /* MinGw GCC 5.3.0 isnan / isinf macros are not working properly, causing a conversion
  * warning / error
  */
-#ifdef __MINGW32__
+#if defined(__MINGW32__) || (defined(__GNUC__) && __GNUC__ >= 5)
```

Comment:
I think if you look at the Gnu C version, then you can (and actually should) leave out the check for mingw. I was wondering about this; it stands to reason that Gnu C (mis-)behavior would be the same across platforms.